src/FrontendAccountCreation.Core.UnitTests/AccountMapperTests.cs

```diff
@@ -108,6 +108,50 @@ public void testCreateAccountModel_AsCompaniesHouseCompany_ShouldReturnAccountMo
         Assert.AreEqual("KAINOS SOFTWARE LIMITED", accountModel.Organisation.Name);
     }
 
+    [TestMethod]
+    public void CreateAccountModel_AsCompaniesHouseCompany_When_AddressIsNull_ShouldReturnAccountModelSuccessfully()
+    {
+        const string json = @"
```

Comment:
You should be able to use the new """ raw string syntax, it will make the json easier to read

Reply:
Done